SQF/dayz_code/actions/maintain_area.sqf

```diff
@@ -37,6 +37,8 @@ if (_count == 0) exitWith {
 };
 
 _requirements = [];
+_option = _this select 0;
```

Comment:
FYI the parameters passed to scripts from addAction are always `_this select 3` so this will error. It's not the same as Zupa's which is just passing a regular array from the dialog button click. 

No worries though, I'm going to make some other changes, so I will fix it.